packages/neuron-wallet/src/controllers/api.ts

```diff
@@ -781,6 +781,13 @@ export default class ApiController {
       }
     })
 
+    handle('get-ckb-node-data-need-size', () => {
+      return {
+        status: ResponseCode.Success,
+        result: Math.ceil(+process.env.CKB_NODE_DATA_SIZE! * 1.2),
```

Comment:
`Math.ceil(+process.env.CKB_NODE_DATA_SIZE! * 1.2)` has been used at https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/controllers/api.ts#L770 and we can assign it to a variable, or define `1.2` as a constant.